package.json

```diff
@@ -84,7 +84,8 @@
     "react-window": "^1.8.5",
     "scplus-shared-components": "https://npm-public.selectquotelabs.com/scplus-shared-components/7.3.1",
     "text-mask-addons": "^3.8.0",
-    "use-debounce": "^6.0.0"
+    "use-debounce": "^6.0.0",
+    "yup": "^0.32.9"
```

Comment:
FWIW, there was a breaking change in Yup 0.31 that was "array().required() will no longer consider an empty array missing and required checks will pass." which we rely on for some of our components, name checkboxgroup. This is why your tests and ci are failing. So I think we need to ugprade yup in a seperate issue which will broke this and #395

Reply:
#398 has been merged, so after a rebase/merge this branch's CI failures should be resolved.

Reply:
I am trying to merge latest changes from issue/244-EnableTypeScript branch, but it is giving me error which is I am unable to understand/resolve. 
![image](https://user-images.githubusercontent.com/16385157/129884256-88e03540-78be-48c8-ab53-8f659ad83b44.png)
Is anyone else facing the same issue while commiting or merging the code or is it only me?

Reply:
We've had this discussion before in Slack somewhere. To get around this, because it's an error you can't really fix without changing config and it's not something that we expect to see often, just do `git commit -m "whatever your commit message is here" --no-verify`

DO NOT skip verification otherwise with `--no-verify`.

Reply:
Looks like a lot of files got touched by prettier or the linter for some reason

Reply:
Yes, I have created a new branch and pushed my changes to that branch "issue/346-EnableTypeScript-SQForm". Maybe creating new pull request and declining this is a good idea.

Reply:
uh, idk about that. Seems unnecessary, imo. bUt that's just me.

Reply:
Yes, you are right, those changes don't seem harmful. So I am sticking with this PR.